### PR TITLE
Wtc 13 welcome page

### LINKED ISSUE #13: Child user story #10 - welcome page
As a child I would like the be given a tablet with a questionnaire that is designed for children and welcomes me to CAMHS, so that I feel like I am an important part of the process. 

Additional technologies on top of HTML, CSS and JavaScript: 
- learn how to implement swiping for Ipad
   - hammer.js
  - greensock
  - jQuery (swipe event listeners or touchSwipe)
- SVG images and animations/transitions

### PR DESCRIPTION
**Pull request by skibinska**
fixes #13 

- add basic file structure
- create and style welcome page
- add fonts, colours
- add lion to the page
- remove unnecessary BEM class
- remove duplicate property from body

### CODE REVIEW

File: main.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  color: #fff;
+  padding: 0px 100px;
+}
+.welcome__title {
+  font-family: 'Henny Penny', cursive;
+  font-size: 4rem;
+  margin: auto;
+  letter-spacing: 0.2rem;
+}
+.welcome__title__organization {
```
Comment (des-des) [inline main.css]: `Block__element__element`? I do not think you are meant to nest names like this in bem

File: main.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  text-align: center;
+  display: block;
+  margin-top: -45px;
+}
+.welcome__image {
+  margin: 25px auto;
+  height: 500px;
```
Comment (des-des) [inline main.css]: Does this work / need to work on mobile?

**Comment by des-des** [comment]
also @skibinska its good to add issue numbers to commit messages

File: main.css (hunk 2 above)
Comment (skibinska) [inline main.css]: No, this web will be build tablet first, as this is the technology most commonly used by the therapists at CAMHS.

**Comment by des-des** [comment]
@skibinska ie
remove duplicate property from body #13

**Pull request merged**